manifests/v/ViGEm/HidHide/1.2.128.0/ViGEm.HidHide.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with WinGet Releaser v2 using Komac v1.5.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ViGEm.HidHide
+PackageVersion: 1.2.128.0

```

```suggestion
PackageVersion: 1.2.128
```
Must match Apps&Features / Control Panel